index.rst

```diff
@@ -5,77 +5,331 @@
+Interface
+---------
+
+The following is an initial proposal for the (xml) interface of the ``Catcher`` and may be revised later.
+
+Commands
+^^^^^^^^
+
+``mute``
+
+   Mute this instance of the ``Catcher``.
+   It will continue to process the data stream and generating reports, but will not issue new alarm unless they have severity higher than the one specified.
+
+   Contains the following parameters:
+
+   ``severity``
+
+      Severity to mute alarms.
+      Alarms will only be issued if severity is higher than this value.
+   
+   ``duration``
+
+      How long to mute?
+   
+   ``mutedBy``
+
+      Person who muted the alarm.
+
+   ``reason``
+
+      Reason alarm is being muted.
+
+``acknowledge``
+
+   Acknowledge alarm.
+
+   Contains the following parameters:
+
+   ``acknowledgedBy``
+
+      Person acknowledging the alarm.
+
+   ``id``
+
+      Id of the process to acknowledge.
+
+``showAnalysisProcess``
+
+   Publish updated information about the analysis processes.
+
+``showAlarms``
+
+   Publish alarm information.
+
+Events
+^^^^^^
+
+``alarm``
+
+   Alarm issued when an analysis process results in a severity level above configured threshold.
+
+   ``severity``
+   
+      Severity of the alarm.
+   
+   ``reason``
+
+      Reason for the alarm to be issued.
+   
+   ``reportUrl``
+
+      Url of the report generated by the analysis process.
+   
+   ``processId``
+
+      Unique identifier for the process that resulted in this alarm.
+   
+   ``acknowledged``
+
+      Was this alarm acknowledged?
+   
+   ``acknowledgedBy``
+
+      Last person who acknowledged the alarm.
+   
+   ``acknowledgedAt``
+
+      Time when the alarm was last acknowledged.
+
+``analysisProcessSummary``
+
+   Summary of the analysis processes.
+
+   ``executing``
+
+      Number of processes currently executing.
+
+   ``executingIds``
+
+      List with the ids of processes currently executing.
+
+   ``finished``
+
+      Number of processes that finished.
+
+   ``finishedIds``
+
+      List with the ids of finished processes.
+
+
+``analysisProcess``
+
+   Information about an analysis process.
+
+   ``id``
+
+      Id of the process.
+
+   ``status``
+
+      Status of the process.
+   
+   ``startedAt``
+
+      When the process started.
+   
+   ``finishedAt``
+
+      When the process finished.
+   
+   ``result``
+
+      Result of the analysis process.
+
+
+Configuration
+-------------
+
+The ``Catcher`` configuration will contain the configuration for all the rules and the CSC picks up a configuration based on its index.
+The following is an example of having 2 instances of the ``Catcher``, with index 1 and 2.
+The configuration is inspired by the use-cases presented in  the `FAFFv2 <https://sitcomtn-037.lsst.io/#deliverable-7-catcher-development>`__ report.
+
+.. code-block:: yaml
 
-.. figure:: /_static/Catcher.png
-   :name: fig-catcher
-   :target: ../_images/Catcher.png
-   :alt: Catcher Architecture.
+   1:
+      rule:
+         monitor:
```

Comment:
I think it would be more informative if there was a bit more logic here.
Say, the wind is above some speed AND from a given direction.
The idea is to show how logic fits into the config.